### PR TITLE
Minor fixes and changes

### PR DESCRIPTION
**Pull request by brec9824**
Patch that contains 8 minor changes/fixes

1. Change the cover command button text when unlocking from showing opening or closing to unlocking.
2. Remove unneeded sensors from the devices table
3. Automatically close the stale image alert when the device uploads a new image
4. Fix stale image alert not appearing when switching sites and locations
5. Create and implement a validation rule for names
6. Remove extra padding after device names/status on the device table on the dashboard
7. Update DeviceController edit() to allow for single field updating (Used for ajax command updates)
8. Center the device image on the dashboard

### CODE REVIEW

File: app/Http/Controllers/DeviceController.php [hunk 1]
```diff
@@ -106,35 +106,46 @@ public function edit($id)
     public function update(EditDevice $request, $id)
     {
         $device = Device::findOrFail($id);
+        $site_id = null;
```
Comment (brandonjank) [inline app/Http/Controllers/DeviceController.php]: Variable $site_id it not used before re-assignment.

File: app/Http/Controllers/DeviceController.php [hunk 2]
```diff
@@ -106,35 +106,46 @@ public function edit($id)
...
+        $location_id = null;
```
Comment (brandonjank) [inline app/Http/Controllers/DeviceController.php]: Variable $location_id is not used before re-assignment.

File: app/Http/Controllers/DeviceController.php (hunk 1 above)
Comment (brec9824) [reply app/Http/Controllers/DeviceController.php]: Fixed #dddaf65

File: app/Http/Controllers/DeviceController.php (hunk 2 above)
Comment (brec9824) [reply app/Http/Controllers/DeviceController.php]: Fixed dddaf65

**Comment by brandonjank** [comment]
@brec9824 Looks like the old approval was removed by you, did you want me to review it again?

**Comment by brec9824** [comment]
@brandonjank Yeah, I guess pushing any changes removes approvals.

**Pull request merged**